src_py/apiServer/apiServer.py

```diff
@@ -41,19 +31,47 @@ def getQueueData(self):
         
         while not received:
             if not multiProcQueue.empty():
-                print("Loss map created!")
-                lossMap = multiProcQueue.get()
-                print(lossMap)
-                return lossMap      
+                print("New loss map created!")
+                multiProcQueue.get()
+                received = True
             time.sleep(0.1)
    
     def train(self):
+        globe.lossMaps.append({})
```

Comment:
We need an option to reset the entire experiments 
reset all lists of result. 
This reset option should be supported by mainServer as well 
@dordor7 @kapelnik